pkg/operator/ceohelpers/unsupported_override.go

```diff
@@ -0,0 +1,26 @@
+package ceohelpers
+
+import (
+	"bytes"
+	"encoding/json"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog"
+)
+
+func IsUnsupportedUnsafeEtcd(spec *operatorv1.StaticPodOperatorSpec) (bool, error) {
+	unsupportedConfig := map[string]interface{}{}
+	if err := json.NewDecoder(bytes.NewBuffer(spec.UnsupportedConfigOverrides.Raw)).Decode(&unsupportedConfig); err != nil {
+		klog.V(4).Infof("decode of unsupported config failed with error: %v", err)
+		return false, err
+	}
+
+	value, found, err := unstructured.NestedBool(unsupportedConfig, "useUnsupportedUnsafeNonHANonProductionUnstableEtcd")
```

Comment:
since there is no typing here to my knowledge on this value, seems like assuming bool might end up being problematic. IE.

```
{"useUnsupportedUnsafeNonHANonProductionUnstableEtcd": "true" }
```

Should we accept any value besides string "false" and `false` as true?

Reply:
addressed